mesop/runtime/context.py

```diff
@@ -1,12 +1,23 @@
 from typing import Any, Callable, Generator, TypeVar, cast
 
+from absl import flags
+
 import mesop.protos.ui_pb2 as pb
 from mesop.dataclass_utils import (
   serialize_dataclass,
   update_dataclass_from_json,
 )
 from mesop.exceptions import MesopDeveloperException, MesopException
 
```

Comment:
Ok, I set to it true to by default. One reason I set it to false was mainly auto caution in case there was some edge case bug we didn't catch. But I guess it's probably better to enable it since most people wouldn't know to enable it in the first place.